modules/test/playwright/tests/analytics-settings-web/utils/analyticsSettings.ts

```diff
@@ -111,6 +111,45 @@ export async function syncAnalyticsCloud({
 	};
 }
 
+export async function syncContactsData({
+	organizationName,
+	page,
+	userGroupName,
+}: {
+	organizationName?: string;
+	page: Page;
+	userGroupName?: string;
+}) {
+	const selectContactsCollapsed = page
+		.locator('[aria-expanded="false"]')
```

Comment:
@ViniLopes20, this seems good, but do we have a more specific selector we can use? I think this one is too generic :sweat_smile: 
If we don't, it's ok then :)

Reply:
Hey @marcosapmf, In this screen we have two Collapsed panels and what differentiates them is only the Text and the value of `aria-expanded`. 

Should I put some locator to get a class that demonstrates that we are looking at these two panels and then see the `aria-expanded`?

Reply:
hmmm do we have any location in an outer context that lets clear what we are looking at?